math-functions.cabal

```diff
@@ -67,6 +69,11 @@ flag system-expm1
      default:     False
      manual:      True
 
+flag system-erf
+     description: Use erf and erfc provided by system. Has no effect with ghcjs
```

Comment:
It would be better to say version defined in library is used with GHCJS. Currently it's not entirely clear what happens

Reply:
Do you have a concrete proposal what it should say? I try to avoid back-and-forth revisions.

Reply:
I tried to write something clearer, but not sure if I made it just worse.